runner/webserver.ts

```diff
@@ -63,10 +63,10 @@ const SAUCE_PORTS = [
  * It provides a static HTTP server for serving the desired tests and WCT's
  * `browser.js`/`environment.js`.
  */
-export function webserver(wct: Context) {
+export function webserver(wct: Context): void {
```

Comment:
Doesn't it seem like it'd be a little nicer if plugins were just classes (a plug-in module exported a constructor) and all the hooks were just optional methods on the class that WCT called into?